sdk/python/tests/compiler/testdata/tolerations.log

```diff
@@ -0,0 +1,3 @@
+[download : main] Connecting to localhost:5678 (127.0.0.1:5678)
+[download : main] wget: can't connect to remote host (127.0.0.1): Connection refused
```

Comment:
This is a test that's designed to be failed. The point of this test is to show it only we can add the toleration to the pod.

Reply:
if you want we can put it to the list of ignored yaml

Reply:
I excluded it for now